packages/embeds/embed-core/src/embed-iframe.ts

```diff
@@ -334,11 +375,11 @@ export const useEmbedType = () => {
 };
 
 function unhideBody() {
+  if (document.body.style.visibility !== "visible") {
+    document.body.style.visibility = "visible";
+  }
```

Comment:
unhideBody immediately. Don't delay to next cycle as that causes a flash of the embedding webpage when loader goes away.